rust_sodium-sys/build.rs

```diff
@@ -248,6 +248,21 @@ fn main() {
         ""
     };
 
+    // Disable PIE for Ubuntu < 15.04 (see https://github.com/jedisct1/libsodium/issues/292)
+    let disable_pie_arg = match Command::new("lsb_release").arg("-irs").output() {
+        Ok(id_output) => {
+            let stdout = String::from_utf8_lossy(&id_output.stdout);
+            let mut lines = stdout.lines();
+            if lines.next() == Some("Ubuntu") {
+                let v = unwrap!(unwrap!(unwrap!(lines.next()).split('.').next()).parse::<u32>());
+                if v < 15 { "--disable-pie" } else { "" }
```

Comment:
it's only until 15.04 i think - and ok on 15.10. Or do we just say below 16.04 we disable-pie ? Happy to merge otherwise.